eldritch/expansions/seaside/abilities.py

```diff
@@ -144,3 +144,40 @@ def get_trigger(self, event, owner, state):
     if isinstance(event, events.Mythos) and event.is_done():
       return events.DiscardSpecific(owner, [self])
     return None
+
+
+class ThickSkulledCombat(events.Combat):
+  def resolve(self, state):
+      # Horror check
+    if (
+        self.monster.difficulty("horror", state, self.character) is not None
+        and self.horror is None
+        and (self.evade is not None or self.combat is not None)
+        and any([self.evade.evaded is False, self.combat.defeated is False])
+    ):
+      self._setup_horror(state)
+    if self.horror is not None:
+      return_early = self._do_horror(state)
+      if return_early:
+        return
+
+    # Combat or flee choice.
+    self._do_combat_or_evade(state)
+
+
+class ThickSkulled(assets.Asset):
+  def __init__(self):
+    super().__init__("Thick Skulled")
+
+  def get_interrupt(self, event, owner, state):
+    if (
+        isinstance(event, events.Combat)
+        and not isinstance(event, ThickSkulledCombat)
+        and event.character == owner
+    ):
+      print("matched")
+      return events.Sequence([
```

Comment:
So here's the list of things that I think we would want to test since we're cancelling the original combat:
- combat with dimensional shambler and/or nightgaunt, which check the event stack and try to cancel the evadeorcombat. might need to have a second monster in the same location to make sure the evadeorcombat is handled properly.
Actually, I think that's it for now. There will likely be more as we add masks, e.g.
http://www.arkhamhorrorwiki.com/The_Bloated_Woman

Reply:
~Nightgaunt~ Dream Flier test aded.